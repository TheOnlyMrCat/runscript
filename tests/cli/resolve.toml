args = ["--color=always", "alpha:run", "bravo:clean", ":test", "charlie", "delta:alt2"]
fs.cwd = "in"
