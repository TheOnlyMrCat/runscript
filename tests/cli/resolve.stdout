Default phase: run
Non-default phase
Phases in separate files
Multiple phases for one target
(Builtin)
(Multiple-custom)
